src/libDirectoryService/DSBlockPreProcessing.cpp

```diff
@@ -658,6 +680,17 @@ VectorOfPoWSoln DirectoryService::SortPoWSoln(const MapOfPubKeyPoW& mapOfPoWs,
       // 5. Finally, sort "FilteredPoWOrderSorter" and stored result in
       // "PoWOrderSorter"
 
+      unsigned int trimmedGuardCount = numNodesTrimmed - (numNodesTrimmed / 3);
```

Comment:
Should we perhaps use the same formula used in `ConsensusCommon::NumForConsensus`?